src/System.Private.CoreLib/shared/System/Math.cs

```diff
@@ -164,22 +164,40 @@ public static double BitIncrement(double x)
 
         public static unsafe double CopySign(double x, double y)
         {
-            // This method is required to work for all inputs,
-            // including NaN, so we operate on the raw bits.
+            const long signMask 
+                = unchecked((long)0b_1000_0000__0000_0000__0000_0000__0000_0000__0000_0000__0000_0000__0000_0000__0000_0000);
 
-            var xbits = BitConverter.DoubleToInt64Bits(x);
-            var ybits = BitConverter.DoubleToInt64Bits(y);
+            if (Sse.IsSupported)
+            {
+                // Create vectors of the elements, and make them float to allow using SSE rather than SSE instructions
```

Comment:
This comment is odd, "...to allow using SSE rather than SSE instructions..." ? (same below in MathF)

Reply:
Oops - meant to say "rather than SSE2" as `andpd` is apparently SSE2

Reply:
Both fixed now :+1: